### PR TITLE
Attempt #6 at removing paver

### PR DESCRIPTION
**Pull request by timmc-edx**
This reverts the previous revert https://github.com/edx/configuration/commit/ce176c4191f98ba9635697a740af28abffbe9644 from https://github.com/edx/configuration/pull/140 so I can add a fix on top of it. See second commit.

---

Make sure that the following steps are done before merging:

  - [x] Have a Site Reliability Engineer review the PR if you don't own all of the services impacted.
  - [x] If you are adding any new default values that need to be overridden when this change goes live, update internal repos and add an entry to the top of the CHANGELOG.
  - [x] Performed the appropriate testing.

### CODE REVIEW

**Review by robrap — APPROVED** [review]
Looks fine, but can you wait until regular business hours to merge and coordinate with anyone or any team who may have run into issues during attempts 1-5? :)

**Comment by timmc-edx** [comment]
Yep! Just queuing it up for tomorrow. :-)

**Pull request merged**
